core/common/buffer.hpp

```diff
@@ -280,6 +280,19 @@ namespace kagome::common {
 
   std::ostream &operator<<(std::ostream &os, const Buffer &buffer);
 
+  namespace literals {
+    /// creates a buffer filled with characters from the original string
+    /// mind that it does not perform unhexing, there is ""_unhex for it
+    inline Buffer operator""_buf(const char *c, size_t s) {
+      std::vector<uint8_t> chars(c, c + s);
+      return Buffer(std::move(chars));
+    }
+
+    inline Buffer operator""_hex2buf(const char *c, size_t s) {
+      return Buffer::fromHex(std::string_view(c, s)).value();
```

Comment:
That's an unsafe method, since it calls .value() on an unchecked result, I doubt we should bring it to the core code.

Reply:
Added static_assert for that